otx/api/usecases/reporting/time_monitor_callback.py

```diff
@@ -67,6 +67,13 @@ def __init__(
 
         self.update_progress_callback = update_progress_callback
 
+    def __getstate__(self):
+        """Return state values to be pickled."""
+        state = self.__dict__
```

Comment:
Shouldn't we copy? `state = self.__dict__.copy()`
BTW, tests are failing due to NoneType error possibly due to this function.
https://github.com/openvinotoolkit/training_extensions/actions/runs/4434177685
Can we just remove this from pickling without actual progress callback funcionality?

Reply:
Thank you for point it out!